fe/fe-core/src/main/java/org/apache/doris/nereids/jobs/load/LoadMgr.java

```diff
@@ -0,0 +1,314 @@
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.nereids.jobs.load;
+
+import org.apache.doris.analysis.CompoundPredicate;
+import org.apache.doris.catalog.Database;
+import org.apache.doris.catalog.Env;
+import org.apache.doris.common.AnalysisException;
+import org.apache.doris.common.CaseSensibility;
+import org.apache.doris.common.DdlException;
+import org.apache.doris.common.LabelAlreadyUsedException;
+import org.apache.doris.common.PatternMatcher;
+import org.apache.doris.common.PatternMatcherWrapper;
+import org.apache.doris.common.util.LogBuilder;
+import org.apache.doris.common.util.LogKey;
+import org.apache.doris.job.common.JobStatus;
+import org.apache.doris.job.extensions.insert.InsertJob;
+import org.apache.doris.job.manager.JobManager;
+import org.apache.doris.load.DataTransFormMgr;
+import org.apache.doris.load.FailMsg;
+import org.apache.doris.load.loadv2.JobState;
+import org.apache.doris.nereids.jobs.load.replay.ReplayLoadLog;
+
+import com.google.common.base.Strings;
+import com.google.common.collect.Lists;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.stream.Collectors;
+
+/**
+ * load manager
+ */
+public class LoadMgr extends DataTransFormMgr {
+    private static final Logger LOG = LogManager.getLogger(LoadMgr.class);
+    private Map<Long, InsertJob> loadIdToJob = new HashMap<>();
+    private Map<String, Long> labelToLoadJobId = new HashMap<>();
+    private Map<Long, Map<String, List<InsertJob>>> dbIdToLabelToLoadJobs = new ConcurrentHashMap<>();
+
+    private JobManager<InsertJob, ?> getJobManager() {
+        return Env.getCurrentEnv().getJobManager();
+    }
+
+    /**
+     * add load job and add tasks
+     * @param loadJob job
+     */
+    public void addLoadJob(InsertJob loadJob) throws DdlException {
+        writeLock();
+        try {
+            if (labelToLoadJobId.containsKey(loadJob.getLabel())) {
+                throw new LabelAlreadyUsedException(loadJob.getLabel());
+            }
+            unprotectAddJob(loadJob);
+            Env.getCurrentEnv().getEditLog().logLoadCreate(ReplayLoadLog.logCreateLoadOperation(loadJob));
```

Comment:
In `unprotectAddJob->registerJob()`, there is an edit log.
And here is another editlog.
I think we should merge these 2 logs